### PR TITLE
using targets to avoid parallel execution in powershell project

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

**Comment by jonsequitur** [comment]
Does this solve the local build problem?

**Comment by colombod** [comment]
No, I can’t see the module folder in the output of the build

**Comment by brettfo** [comment]
The `DependsOnTargets="Restore"` will just move the slowness to later in the build.  I had some luck locally simply wrapping the entire `<ItemGroup>` with `<Target Name="GatherModulesContent" BeforeTargets="Build">`.

Can you try that locally to see what happens?

File: Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.csproj [hunk 1]
```diff
@@ -31,39 +31,43 @@
              Link="Modules\%(RecursiveDir)%(FileName)%(Extension)"
              PackagePath="contentFiles/any/any/Modules"
              PackageCopyToOutput="true"
-             CopyToOutputDirectory="PreserveNewest" />
+             CopyToOutputDirectory="PreserveNewest"/>
```
Comment (brettfo) [inline Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.csproj]: This hurts.

**Pull request merged**